amacc.c

```diff
@@ -529,6 +529,9 @@ void stmt()
         else fatal("semicolon expected");
         return;
     case For:
+        /* For loop is implemented as:
```

Comment:
Use C++ style comments `//` rather than C-style since AMaCC doesn't support the later at present.